lib/bolt/config/options.rb

```diff
@@ -260,6 +260,10 @@ module Options
                     description: "The name of the module.",
                     type: String
                   },
+                  "resolve" => {
+                    description: "Whether to resolve the module's dependencies when installing modules.",
+                    type: [TrueClass, FalseClass]
```

Comment:
Hm, this weirdly doesn't seem to error with non-boolean values are provided. Do we not run the validator when doing `module install`?

Reply:
No, the validator would need to be updated to handle the `oneOf` key since there are multiple valid definitions for a module spec. I can add in a separate check as part of the module objects to ensure that it's a boolean.

Reply:
That'd be great.

Reply:
Actually, it seems like the validator should be checking every key that's there against it's type even if there are multiple possible configurations (basically should be able to handle `oneOf`). It looks like this is the only instance of `oneOf`, so it might be appropriate to add here.